backend/module/sourcecontrol/sourcecontrol.go

```diff
@@ -31,22 +42,101 @@ func New(*any.Any, *zap.Logger, tally.Scope) (module.Module, error) {
 		return nil, errors.New("service was not the correct type")
 	}
 
-	return &mod{c}, nil
+	return &mod{c, config}, nil
 }
 
 type mod struct {
 	github github.Client
+	config *sourcecontrolconfigv1.Config
 }
 
 func (m *mod) Register(r module.Registrar) error {
 	sourcecontrolv1.RegisterSourceControlAPIServer(r.GRPCServer(), m)
 	return r.RegisterJSONGateway(sourcecontrolv1.RegisterSourceControlAPIHandler)
 }
 
-func (m *mod) CreateRepository(ctx context.Context, req *sourcecontrolv1.CreateRepositoryRequest) (*sourcecontrolv1.CreateRepositoryResponse, error) {
-	return m.github.CreateRepository(ctx, req)
+func contains(s []string, str string) bool {
+	for _, v := range s {
+		if v == str {
+			return true
+		}
+	}
+	return false
+}
+
+func (m *mod) ManageableOrganizations(ctx context.Context) ([]*sourcecontrolv1.Entity, error) {
+	organizations, err := m.github.ListOrganizations(ctx, github.CurrentUser)
+	writableOrgs := []*sourcecontrolv1.Entity{}
+	if err != nil {
+		return nil, err
+	}
```

Comment:
i think this is what Mike was referring to
```suggestion
	organizations, err := m.github.ListOrganizations(ctx, github.CurrentUser)
	if err != nil {
		return nil, err
	}
	writableOrgs := []*sourcecontrolv1.Entity{}
```